netlify.toml

```diff
@@ -0,0 +1,3 @@
+[functions.ssr]
+  node_bundler = "esbuild"
+  included_files = ["dist/apps/storefront/**"]
```

Comment:
not enough:
- redirect is needed to ensure all deep links redirect to index.html
- security headers would be nice to have here, as it will help us to understand potential integration issues upfront while we're building features

Reply:
Added missed files and package